core/src/main/java/org/apache/iceberg/rest/CatalogHandlers.java

```diff
@@ -170,16 +171,20 @@ public static LoadTableResponse stageTableCreate(Catalog catalog, Namespace name
     properties.putAll(request.properties());
 
     String location;
+    String metadataLocation;
     if (request.location() != null) {
       location = request.location();
+      metadataLocation = location + "/metadata";
```

Comment:
Why does this hard-code `/metadata`?

Reply:
Is there a particular constant you’d like me to use or to skip adding metadata location for this section where only `location` is known?

Reply:
I'll just skip location in this case as it doesn't particularly matter as we're not making it a required field.